admin/index.php

```diff
@@ -26,7 +26,7 @@
 	<link rel="stylesheet" type="text/css" href="../resources/css/admin.css" />
 </head>
 <body class="deselect">
-<div id="wrapper" style="overflow-y: auto;">
+<div id="wrapper" style="overflow-y: auto; background-image: url('<?=$config['background_admin']?>');">

```

Comment:
I think it would be nice to use css vars like we do for the color schema. Therefore the html stays as clean as possible.

Reply:
Ok, will take a look next days

Reply:
Should I rename `_colors.scss` to `_theme.scss` and place it there ?